start.go

```diff
@@ -75,7 +76,7 @@ func StartAndDialContext(ctx context.Context, opts ...Option) (conn Connection,
 
 func writeDefaultConfigFiles(ctx context.Context, o *option) error {
 	configFiles := map[string]string{
-		vppConfFilename: fmt.Sprintf(o.vppConfig, o.rootDir),
+		vppConfFilename: fmt.Sprintf(o.vppConfig, o.rootDir, o.additionalStanza),
```

Comment:
I looks like you are basically just 'appending' config to the default config, correct?

I am curious, why don't you just stomp the whole config using WithVPPConfig?

Reply:
@ftehlar Yeah... let me know... I am totally open to other ideas... and there may be reasons that haven't occurred to me why WithVPPConfig is the wrong answer for you... but I couldn't think of any off the top of my head :)